webscraper/dataScraperAPI.js

```diff
@@ -1,6 +1,6 @@
 function scrape() {
```

Comment:
Refactor your file structure to clearly separate frontend from backend (e.g., 'web' folder and 'server' folder). Within the server folder, place the webscaperAPI file and all other backend related files. Each folder should have its own package.json and package-lock.json. It's helpful to separate this for production environments, as they both "live" or "operate" in different environments (e.g., browser for frontend).